### DISCUSSION TITLE
Can ACEMD be used with OpenPathSampling?

### DISCUSSION THREAD

**Discussion by varun-go** [openpathsampling/openpathsampling#1144] [Q&A]
Is it possible to implement ACEMD with OPS? I am asking in the context of using NNP/MM models with OPS. The paper "[NNP/MM: Accelerating Molecular Dynamics Simulations with Machine Learning Potentials and Molecular Mechanics](https://doi.org/10.1021/acs.jcim.3c00773)" uses OpenMM and OpenMM-Torch through ACEMD.

Since OPS supports OpenMM, can ACEMD also be used (since it appears to use OpenMM as the MD engine)? 

Thank you.

**Comment by sroet** [comment]
Hey @varun-go,

Thanks for the question: first a warning, neither me nor @dwhswenson (the currently listed maintainers) are currently paid to maintain this package and therefor the support will be minimal.


It should be possible to implement, but I don't see any reference to source code when looking at the [ACEMD docs](https://software.acellera.com/acemd/manual.html) and at their [github organisation](https://github.com/Acellera). This means that we can't implement ACEMD as we did with OpenMM (with direct calls to their source code), instead it should probably be implemented [as was done for GROMACS](https://github.com/openpathsampling/openpathsampling/blob/master/openpathsampling/engines/gromacs/engine.py).

We always welcomes PRs if you want to make one that adds ACEMD to OPS

**Reply by varun-go** [reply]
Thank you for getting back to me.

After some consideration, I think I will pursue using the software with OpenMM or GROMACS.
